keysign/app.py

```diff
@@ -171,61 +174,54 @@ def on_activate(self, app):
 
         window.show_all()
         self.add_window(window)
-
+        reactor.run()
```

Comment:
In fact it was a bit strange.

Fortunately this was not the correct way to do it, when I wrote it I missed a line in the documentation.
I fixed it with the commit cc9fc00
With registerGApplication twisted can handle the Gtk mainloop, now it is way cleaner.